exchanges/account/account.go

```diff
@@ -0,0 +1,63 @@
+package account
+
+import (
+	"errors"
+
+	"github.com/thrasher-corp/gocryptotrader/currency"
+)
+
+var accounts = make(map[string]*Holdings)
+
+// Holdings is a generic type to hold each exchange's holdings for all enabled
+// currencies
+type Holdings struct {
+	Exchange string
+	Accounts []SubAccount
+}
+
+// SubAccount defines a singular account type with asocciated currency balances
+type SubAccount struct {
+	ID       string
+	Currency []Balance
```

Comment:
I can appreciate this change in the sense that you can say currency is the plural of currency, but we need to be consistent.  We have 925 instances of "currencies" in the our codebase. If you keep this, then please update all of them, otherwise revert

Reply:
Yeah I see your point. I will reverty.